### PR TITLE
Add lint rule and fix a few issues

### PR DESCRIPTION
**Pull request by jeroach**
So I updated the lint rule as per the thread over in Slack:
https://d2l.slack.com/archives/CDJE0AYMB/p1547753191076900

I fixed a few errors, but these 2 are stumping me as of now:
```
> d2l-activities@0.0.2 lint:wc C:\Code\GitHub\WebComponents\activities
> polymer lint -i "./*" "components" "demo" "test"



const ActivityListItemResponsiveConstantsImpl = (superClass) => class extends superClass {
                                                                              ~~~~~~~~~~

components/d2l-activity-list-item/ActivityListItemResponsiveConstants.js(3,79) warning [could-not-resolve-reference] - Could not resolve reference to class


<!doctype html>
~

demo/index.html(1,1) warning [internal-lint-error] - Internal error during linting: Cannot read property '__location' of undefined


Found 2 warnings.
```

Any ideas as to what's causing these? I'll continue to dig. Thanks!

### CODE REVIEW

File: demo/d2l-evaluation-hub/data/activities/activitiesList/filters.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (jeroach) [inline demo/d2l-evaluation-hub/data/activities/activitiesList/filters.json]: These 3 files needed to be valid JSON

File: demo/index.html [hunk 2]
```diff
@@ -13,15 +13,17 @@
...
-		<style is="custom-style" include="demo-pages-shared-styles"></style>
-		<style is="custom-style" include="d2l-typography">
-				html {
-					font-size: 20px;
-				}
-				.fixedSize {
-					max-width: 1600px;
-				}
-		</style>
+		<custom-style>
```
Comment (jeroach) [inline demo/index.html]: `style is="custom-style"` needs to be wrapped in a `<custom-style>` tag

**Comment by m6jones** [comment]
The first problem, I forgot to add `/* @polymerMixin */` above the definition.

**Comment by jeroach** [comment]
Fixed the other block - needed to wrap the `custom-style` in an `$_documentContainer.innerHTML`

**Pull request merged**